src/alr/alr-paths.ads

```diff
@@ -92,7 +92,7 @@ private
    function Alr_Source_Folder return String
    is (OS_Lib.Getenv (Alire.Environment.Source, Alr_Config_Folder / "alire"));
 
-   function Projects_Folder      return String
-   is (Alr_Working_Cache_Folder / "projects");
+   function Releases_Folder      return String
+   is (Alr_Working_Cache_Folder / "releases");
```

Comment:
Why not `"crates"` here?

Reply:
I've realized that many times when we talk about crates we are talking about a particular release of a crate. In this case, we are downloading releases, so that's it. Although it's kind of splitting hairs probably.

Reply:
On further thought, this should probably be better "dependencies"

Reply:
Done in e07df2a